ambassador/ambassador/cli.py

```diff
@@ -112,6 +112,11 @@ def tls_secret_resolver(secret_name: str, context: str) -> Optional[Dict[str, st
             'cacert_chain_file': "/path/to/%s.crt" % secret_name,
         }
     else:
+        if cert_dir is not None:
+            return {
+                'certificate_chain_file': os.path.join(cert_dir, 'cert.crt'),
```

Comment:
OK. Per @containscafeine, the field name in the Envoy context changed in V2. We're going to keep this as-is for EA5 since we need to do some context cleanup shortly after anyway.

Reply:
envoy v2 has renamed the field to `certificate_chain_file` so I figured that would be better for the new IR.